etl/steps/data/garden/tourism/2024-08-17/unwto_gdp.py

```diff
@@ -0,0 +1,36 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("unwto_gdp")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["unwto_gdp"].reset_index()
```

Comment:
if you want, you can replace this with tb = ds_meadow.read_table("unwto_gdp"), which automatically resets the index

Reply:
nice! pretty sure these just get prefilled by wizard but good to know we have this function!